components/gallery/GalleryItem.vue

```diff
@@ -207,6 +183,7 @@ import CarouselTypeRelated from '@/components/carousel/CarouselTypeRelated.vue'
 import CarouselTypeVisited from '@/components/carousel/CarouselTypeVisited.vue'
 import CollectionDetailsPopover from '@/components/collectionDetailsPopover/CollectionDetailsPopover.vue'
 
+import GalleryItemToolBar from './GalleryItemToolBar.vue'
```

Comment:
can be auto imported 

![CleanShot 2024-03-29 at 09 01 16@2x](https://github.com/kodadot/nft-gallery/assets/44554284/40d10d02-7893-4150-ae39-d5e98014ac70)